examples/step-35/step-35.cc

```diff
@@ -451,8 +451,8 @@ namespace Step35
     DoFHandler<dim> dof_handler_velocity;
     DoFHandler<dim> dof_handler_pressure;
 
-    QGauss<dim> quadrature_pressure;
-    QGauss<dim> quadrature_velocity;
+    const QGauss<dim> quadrature_pressure;
+    const QGauss<dim> quadrature_velocity;
```

Comment:
This doesn't look right since this is a declaration.

Reply:
It's a member variable. It's going to be initialized in the constructor, but it can be `const`, no?

Reply:
you're right!